aws_lab_setup/roles/managed_node/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+- name: Configure sshd and sudoers
+  lineinfile:
+    dest: "{{ item.dest }}"
+    regexp: "{{ item.regexp }}"
+    line: "{{ item.line }}"
+    state: "{{ item.state | default('present') }}"
+    validate: "{{ item.validate | default(omit) }}"
+    backup: no
+  with_items: "{{ managed_node_config_options }}"
+  notify: restart ssh
+  tags:
+    - ssh
+    - sudo
+
+- name: Install firewalld and libselinux-python
+  yum:
+    name:
+      - libselinux-python
+      - firewalld
+    state: present
+  when:
+    - "ansible_os_family == 'RedHat' or ansible_os_family == 'CentOS'"
```

Comment:
ansible_os_family will never be "CentOS." were you thinking of a different fact like ansible_distribution?